src/react-apps/applications/dashboard/src/services/servicesOverview.tsx

```diff
@@ -225,12 +222,12 @@ class ServicesOverviewComponent extends React.Component<IServicesOverviewCompone
                 />
               </Grid>
               <Hidden lgUp={true}>
-                {this.renderSort()}
+                {/* {this.renderSort()} */}
                 {this.renderFilters()}
               </Hidden>
               <Hidden mdDown={true}>
                 {this.renderFilters()}
-                {this.renderSort()}
+                {/* {this.renderSort()} */}
```

Comment:
Remove comment?

Reply:
sortbutton will be commented in when sort is implemented